edx_proctoring/tests/test_views.py

```diff
@@ -1773,8 +1773,7 @@ def _setup_for_test_exam_callback(self):
 
         return response_data['exam_attempt_id']
 
-    @patch('edx_proctoring.callbacks.switch_is_active')
-    def test_exam_callback(self, mock_switch_is_active):
+    def test_exam_callback_rpnowv4(self):
```

Comment:
I think this should just stay 'test_exam_callback' since it isn't considered specific to rpnowv4 anymore.